app/Services/Manager.php

```diff
@@ -134,9 +134,13 @@ public function collectContestInfo($data)
      * @param  bool  $withReportback
      * @return \Illuminate\Support\Collection  $users
      */
-    public function getModelUsers($model, $withReportback = false)
+    public function getModelUsers($model, $withReportback = false, $subscribed = false)
     {
-        $users = $model->users;
+        if (! $subscribed) {
```

Comment:
Think it would be cool to use a [ternary](https://davidwalsh.name/php-shorthand-if-else-ternary-operators) here:

`$users = ! $subscribed ? $model->users : $model->subscribers;`